docs/app/documentation/component-docs/dropdown/examples/dropdown-default-example.component.html

```diff
@@ -1,5 +1,5 @@
 <fd-popover style="margin-right: 12px;">
-    <fd-dropdown-control>Compact Dropdown</fd-dropdown-control>
+    <fd-dropdown-control>Cozy Dropdown</fd-dropdown-control>
```

Comment:
Keep `Compact`.

Reply:
Should be `Cozy`, shouldn't it? Both examples say `Compact` when one should be `Compact` and the other should be `Cozy`

Reply:
Please see screenshot for difference size of dropdown
![Screen Shot 2019-08-22 at 10 20 18 AM](https://user-images.githubusercontent.com/41170611/63522507-81195300-c4c6-11e9-8244-2adced7081a6.png)

Reply:
Ok you are completely right.